.github/workflows/rspec_rubocop.yml

```diff
@@ -0,0 +1,77 @@
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on: [pull_request, push]
+
+permissions:
+  contents: read
+
+env:
+  CC: ccache gcc
+  CXX: ccache g++
+  THRIFT_SHA256SUM: 2289d02de6e8db04cbbabb921aeb62bfe3098c4c83f36eec6c31194301efa10b  thrift-0.10.0.tar.gz
+  THRIFT_URL: http://archive.apache.org/dist/thrift/0.10.0/thrift-0.10.0.tar.gz
+  THRIFT_FILE: thrift-0.10.0.tar.gz
+  THRIFT_DIR: thrift-0.10.0
+  DEBIAN_FRONTEND: noninteractive # for apt to be noninteractive
```

Comment:
This is needed because otherwise it will get stuck when configuring `tzdata`:

```
#6 141.7 Setting up tzdata (2022g-0ubuntu0.20.04.1) ...
#6 141.8 debconf: unable to initialize frontend: Dialog
#6 141.8 debconf: (TERM is not set, so the dialog frontend is not usable.)
#6 141.8 debconf: falling back to frontend: Readline
#6 141.9 Configuring tzdata
#6 141.9 ------------------
#6 141.9
#6 141.9 Please select the geographic area in which you live. Subsequent configuration
#6 141.9 questions will narrow this down by presenting a list of cities, representing
#6 141.9 the time zones in which they are located.
#6 141.9
#6 141.9   1. Africa      4. Australia  7. Atlantic  10. Pacific  13. Etc
#6 141.9   2. America     5. Arctic     8. Europe    11. SystemV
#6 141.9   3. Antarctica  6. Asia       9. Indian    12. US
#6 141.9 Geographic area:
```